pull request de bora bill amostradinho casca de bala da silva

gabrieldodrip990 opened the pull request:
![9b790ccc1c822d24bdd69122a2a1777dcfe23cee70907c8d2ae93ec0da5f2d77_1 jpg](https://github.com/user-attachments/assets/f71e8284-9a3f-4eaa-8ae4-7039b07d4a39)

gabrieldodrip990 committed:
adicionar mapa

gabrieldodrip990 committed:
musica

Cachorro112 commented:
você precissa enviar tambem o Json de cada mapa para eles funcionarem. favor colocar///grato

gabrieldodrip990 commented:
Ok, pera ai

Cachorro112 commented on _maps/_basemap.dm:
```diff
@@ -24,7 +24,11 @@
+		// MATAGAL EDIT
+		#include "effigy\map_files\FoxHoleStation\foxholestation.dmm"
+		#include "effigy\map_files\RimPoint\RimPoint.dmm"
+		#include "effigy\map_files\SigmaOctantis\SigmaOctantis.dmm"
```
Acho que o Rimpoint e SigmaOctantis podem ser retirados já que ambos são mapas muito grandes e não seriam eficiente no server. ao tirar eles tambem seria necessario tirar suas ruins.

gabrieldodrip990 committed:
tinha esquecido isso

gabrieldodrip990 commented:
resolvido

gabrieldodrip990 committed:
consertando erros

arrumei algumas coisas que estavam voltando no novo mapa, fiz com que ficasse **funcional**, consertei alguns bugs do foxholestation, os outros 2 não são essenciais por serem bastante grandes para a pop atual e provavelmente estão bugados tambem

Cachorro112 reviewed (approved):
Aprovado. apenas o foxhole será usado por enquanto.
